matchengine/me_market.c

```diff
@@ -775,26 +775,11 @@ static int execute_market_bid_order(bool real, market_t *m, order_t *taker)
 
         order_t *maker = node->value;
         mpd_copy(price, maker->price, &mpd_ctx);
-
```

Comment:
(More of a stylistic and lower priority comment than anything else.)

For a file of this size, could we not replace many of the if/else blocks with a ternary operator, to improve readability?